src/cookbooks/data_modeling/shredder_mitigation.md

```diff
@@ -0,0 +1,175 @@
+# Shredder mitigation during backfills
+
+Running a backfill on aggregate tables at Mozilla comes with the high risk of altering reported metrics due to the effects of [shredder](https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/6848694/Shredder). It is essential for the business to mitigate these risks while still allowing the addition and modification of columns in aggregate tables to enhance analytic capabilities or propagate changes in business definitions. At the same time, this mitigation must happen in alignment with [Mozilla data protection policies](https://www.mozilla.org/en-US/privacy/firefox/) by preventing aggregate tables from maintaining data that could be used to uniquely identify shredded clients.
+
+**Shredder mitigation** is a process that breaks through this risk by securely executing backfills on aggregate tables. It recalculates metrics using both existing and new data, mitigating the impact of shredder and propagating business definition changes while maintaining data integrity over time.
+
+This documentation provides step-by-step guidance with example and common scenarios, to help you achieve consistent results.
+
+For context and details on previous analysis and discussions that lead to this solution, refer to the [proposal](https://docs.google.com/document/d/1lU7qSjAOFrdcQuDbdhFzJLgP6Mkn6L8XBjRtFdq6qkU/edit?usp=sharing).
+
+[//]: # "<!-- TOC -->"
```

Comment:
Not sure about the intention of this piece of syntax here.
If you wanted this to be auto-filled with a table of contents, you need just this:

```suggestion
<!-- toc -->
```

Reply:
Yes, it was pending to generate the TOC, thanks!